karate-tests/src/test/java/patients/patientAccess/getPatient.feature

```diff
@@ -49,6 +49,7 @@ Feature: Patient Access (Retrieve)
       | patientType | nhsNumber   |
       | P0          | 9912003073  |
       | P5          | 9912003072  |
+      | P9.Cp       | 5900068196  |
```

Comment:
[NHS Login test users](https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?spaceKey=APM&title=Testing+with+mock+auth#Testingwithmockauth-NHSLogintestusers) mentions a few P9s with no VOT, would be good to cover that scenario as well